Core/Interfaces/BasicCrudServices/IRead.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Core.Entities;
+using System;
+
+namespace Core.Interfaces.BasicCrudServices
+{
+    public interface IRead<T>
```

Comment:
IDE mi podpowiada, że można zadeklarować parametr T jako covariant (zapis: `out T`) - nie wiem czy jest nam to potrzebne, warto sprawdzić